AmplifyPlugins/Auth/Tests/AWSCognitoAuthPluginUnitTests/ResolverTests/AuthorizationState/FetchAuthSessionStateResolverTests.swift

```diff
@@ -27,7 +27,7 @@ extension FetchAuthSessionStateSequence {
 class FetchAuthSessionStateResolverTests: XCTestCase {
 
     func testValidFetchAuthSessionStateSequences() throws {
```

Comment:
we can remove this test class altogether if it's not needed

Reply:
I'll defer to @royjit on this.  They may update and use this as they complete the Auth work.